server/src/models/schema_root.js

```diff
@@ -10,7 +10,7 @@ export default {
     }
 
     type Query {
-      root(lang:String!): Root
+      root(lang:String!): Root!
```

Comment:
Making root non-nullable is going to avoid extra optional chaining once types are generated. 
```ts
// (Before) with a nullable root field 
const services = data?.root?.org?.services
// (After) with a  non-null root
const services = data?.root.org?.services
```